packages/openapi-generator/src/file-serializer/api-file.ts

```diff
@@ -0,0 +1,59 @@
+import { codeBlock, unique } from '@sap-cloud-sdk/util';
+import { OpenApiApi, OpenApiOperation } from '../openapi-types';
+import { collectRefs, hasNotSchema, parseTypeNameFromRef } from '../model';
+import { serializeOperation } from './operation';
+import { Import, serializeImports } from './imports';
+
+export function apiFile(api: OpenApiApi): string {
+  const imports = serializeImports(getImports(api));
+  return codeBlock`
+${imports}
+
+export const ${api.name} = {
+  ${api.operations.map(operation => serializeOperation(operation)).join(',\n')}
+};
+`;
+}
+
+/**
+ * Get the reference types for all request body types in the given operation list.
+ * @param operations The given operation list.
+ * @returns The list of body types.
+ */
+function collectRefsFromOperations(operations: OpenApiOperation[]): string[] {
+  return operations.reduce(
+    (referenceTypes, operation) =>
+      unique([
+        ...referenceTypes,
+        ...collectRefs(operation.requestBody?.schema),
+        ...collectRefs(operation.response)
+      ]),
+    []
+  );
+}
+
+function hasNotSchemaInOperations(operations: OpenApiOperation[]): boolean {
+  return operations.reduce((foundNotSchema, operation) => {
+    const requestBodySchema = operation.requestBody?.schema;
+    return (
+      foundNotSchema || (!!requestBodySchema && hasNotSchema(requestBodySchema))
```

Comment:
I find the boolean here hard to graps. Can you move it down to the hasNotSchema. Also I find the [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method on the array better in a case if you want to check if at least one element fullfills boolean. What do you think?

Reply:
OMG, that's so much better!